README.md

```diff
@@ -82,6 +82,36 @@ You can use `eth-staking-smith` as follows to convert your address:
 
 Note that --validator-index and --validator-start-index are two distinct parameter, the former being index of validator on Beacon chain, and the latter is the index of validator private key derived from the seed
 
+
+### Command to send SignedBLSToExecutionChange request to Beacon node
+
+```
+curl -H "Content-Type: application/json" -d '{
```

Comment:
why are we showing curl command instead of cli cmd?

Reply:
The command right now does not support sending output to beacon node; it should be easy to add though given `openssl-sys` is already a dependency. I will do it in https://github.com/ChorusOne/eth-staking-smith/pull/54 as this PR only touches presigned exit message, but bls-to-execution-change

Reply:
Sending to beacon node was implemented in https://github.com/ChorusOne/eth-staking-smith/pull/54/commits/64ef067bcd8728f5aed27463c2881cf56eadf79b